O/OpenSpiel/build_tarballs.jl

```diff
@@ -2,7 +2,7 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
-julia_version = v"1.6.3"
+julia_version = v"1.7.0"
```

Comment:
You can't possibly change compat bound _and_ keep the same version of the package, this would wreak havoc for v1.6 users.  I think instead you want to follow what other packages do, see for example https://github.com/JuliaPackaging/Yggdrasil/blob/03229dc14370ddac1530e038d671728ec1a7db0b/L/libsingular_julia/build_tarballs.jl

Reply:
Thanks!

I tried to add support 1.6 but encountered the UUID not found error. The trick in the link provided above works very well!